src/nanoemoji/write_glyphmap_for_glyph_svgs.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Generates a glyphmap for svgs named by glyph id."""
+from absl import app
+from absl import flags
+from absl import logging
+from fontTools import ttLib
+from nanoemoji.glyphmap import GlyphMapping
+from nanoemoji import util
+from pathlib import Path
+
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_string("output_file", "-", "Output filename ('-' means stdout)")
+
+
+def main(argv):
+    input_files = util.expand_ninja_response_files(argv[1:])
+    source_font = util.only(lambda a: a.endswith(".ttf"), argv)
```

Comment:
An explicit flag for --source_font would be preferable, but if you want to use the positional arguments, bear in mind, you want to pop the `source_font` out of the `input_files` you just expanded from the ninja .rsp file, not straight from `argv`. Right now this seems to work only because you didn't actually tell ninja to use a rspfile/rspfile_content in the rule definition (see my comment above).

Reply:
My kingdom for Rust where the expand could consume argv! Good catch.